CommunityProjects.md

```diff
@@ -8,30 +8,40 @@ SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
 This page collects seL4-based projects. If you know of a project that
 you'd like to be listed here, add it.
 
+## Application areas
 The seL4 kernel is currently used in projects in the following application areas:
 
 - Automotive
-- Aviation (e.g.,
-        [SMACCM](https://trustworthy.systems/projects/TS/SMACCM/))
+- Aviation
+  - [SMACCM](https://trustworthy.systems/projects/TS/SMACCM)
 - Connected Consumer Devices
 - SCADA
-- Spaceflight (e.g., the
-        [UNSW QB50](https://trustworthy.systems/projects/TS/qb50) satellite.)
```

Comment:
Same here, QB50 is by far not the only space project with seL4. It's one of the few that is public, but this lists application areas and removing "e.g." creates a wrong impression.